components/atoms/Select/single-select.tsx

```diff
@@ -63,7 +63,12 @@ const SingleSelect = ({
           insetLabel && `before:content-[attr(data-inset-label)] before:mr-1 before:font-normal before:text-slate-500`
         )}
       >
-        <p className="grow text-start">{current?.label ?? placeholder}</p>
+        <Tooltip content={current?.label ?? placeholder}>
+          <div className="flex items-center w-[200px]">
```

Comment:
nit: we want to try as much as possible to reduce hard-coded values...
```suggestion
          <div className="flex items-center w-48">
```

Reply:
Thank you @OgDev-01